src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -124,23 +124,13 @@ private static bool ImplementationExists(INamedTypeSymbol classOrStructType, ISy
         private static bool IsInterfacePropertyImplemented(INamedTypeSymbol classOrStructType, IPropertySymbol propertySymbol, CancellationToken cancellationToken)
         {
             // A property is only fully implemented if both it's setter and getter is implemented.
-            if (propertySymbol.GetMethod != null)
-            {
-                if (classOrStructType.FindImplementationForInterfaceMember(propertySymbol.GetMethod) == null)
-                {
-                    return false;
-                }
-            }
 
-            if (propertySymbol.SetMethod != null)
+            return isAccessorImplemented(propertySymbol.GetMethod, classOrStructType) && isAccessorImplemented(propertySymbol.SetMethod, classOrStructType);
+
+            static bool isAccessorImplemented(IMethodSymbol accessor, INamedTypeSymbol classOrStructType)
```

Comment:
small preference to write `// local functions` above the first local function.  it helps clarify that only local functions should follow from that point on.